packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -117,18 +109,20 @@ export default compose( [
 			getCurrentPostType,
 			isCurrentPostPublished,
 			isCurrentPostScheduled,
-			isSavingPost,
+			isEditedPostBeingScheduled,
 			isEditedPostDirty,
+			isSavingPost,
 		} = select( 'core/editor' );
 		const { isPublishSidebarEnabled } = select( 'core/editor' );
 		return {
-			postType: getCurrentPostType(),
 			hasPublishAction: get( getCurrentPost(), [ '_links', 'wp:action-publish' ], false ),
-			isPublished: isCurrentPostPublished(),
-			isScheduled: isCurrentPostScheduled(),
-			isSaving: isSavingPost(),
+			isBeingScheduled: isEditedPostBeingScheduled(),
 			isDirty: isEditedPostDirty(),
+			isPublished: isCurrentPostPublished(),
 			isPublishSidebarEnabled: isPublishSidebarEnabled(),
+			isSaving: isSavingPost(),
+			isScheduled: isCurrentPostScheduled(),
+			postType: getCurrentPostType(),
```

Comment:
It doesn't look like `postType` is used anywhere.

Reply:
2bac72b